build.assets/tooling/cmd/resource-ref-generator/reference/testdata/src/app.go

```diff
@@ -0,0 +1,90 @@
+// Teleport
+// Copyright (C) 2025  Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package typestest
+
+import (
+	fmt "fmt"
```

Comment:
Why?

Reply:
We use `fmt` in `api/types/app.go`, which this file is based on. Not sure where that strange package alias came from.